src/ray/core_worker/core_worker.cc

```diff
@@ -792,10 +792,35 @@ CoreWorker::CoreWorker(CoreWorkerOptions options, const WorkerID &worker_id)
 
   actor_creator_ = std::make_shared<DefaultActorCreator>(gcs_client_);
 
+  const auto dispatch_nccl_send_callback = [this](const ObjectID &object_id, const ActorID &dst_actor_id) {
```

Comment:
The callback function used by the driver to send a RPC `ExecuteNcclSend` to the sender actor to call Nccl send to send object `object_id` to actor `dst_actor_id`.